### ISSUE TITLE
mk-ival: "cannot reference an identifier before its definition"

### ISSUE THREAD

**Issue by postmath** [herbie-fp/rival#6]
To reproduce:
```
% racket --version
Welcome to Racket v8.3 [cs].
% racket
> (require rival math/bigfloat)
> (ival 2.bf 2.bf)
(ival (bf 2) (bf 2))
> (mk-ival 2.bf)
; mk-ival: undefined;
;  cannot reference an identifier before its definition
;   in module: top-level
; [,bt for context]
> ,bt
; mk-ival: undefined;
;  cannot reference an identifier before its definition
;   in module: top-level
;   context...:
;    body of top-level
;    /usr/share/racket/pkgs/xrepl-lib/xrepl/xrepl.rkt:1493:0
;    /usr/share/racket/collects/racket/repl.rkt:11:26
```

I'm afraid I don't speak racket, so I can't figure out if this is just a problem with the racket version, or a true bug in rival. This is racket version `8.3+ppa1-1~bionic1` from https://launchpad.net/~plt/+archive/ubuntu/racket and rival version... I'm not exactly sure, whatever was pulled in by the latest herbie, so that should be v1.4 I think?

**Comment by pavpanchekha** [comment]
Uhh... Wow! I pushed a commit yesterday that broke it. I'll fix it.

**Comment by pavpanchekha** [comment]
This is fixed in 0c37eb2278d336351f05bb36a6a547e6296a35fa, which should be available via `raco pkg update rival` in a few minutes.
